dash/frontend/src/app/core/dialogues/show-json-data/show-json-data.component.ts

```diff
@@ -54,8 +55,19 @@ export class ShowJsonDataComponent implements OnInit {
     this.falcoService.getFalcoLogs(this.clusterId,  { limit: this.limit, page: this.page, signature: this.signature})
       .pipe(take(1))
       .subscribe(response => {
-        this.dataSource = new MatTableDataSource(response.data.list);
-        this.logCount = response.data.logCount;
+        const dataList = response.data.list;
+        // one less log count - without the current event log
+        this.logCount = response.data.logCount - 1;
+
+        // create a new data list without the current event log
+        for (let count = 0; count < this.logCount; count++){
+          this.newDataList = dataList.filter(i => i.id !== this.data.content.id);
+        }
+
+        this.dataSource = new MatTableDataSource(this.newDataList);
+        // clear out previous data
+        this.newDataList = [];
```

Comment:
Why set newDataList above only to clear it out down here?

Reply:
Should this newDataList perhaps be a local variable instead of a class member variable?